### PR TITLE
Feature/need milk test cag

### PR DESCRIPTION
**Pull request by cagallo**
Is this a fix or a feature?
- [ ] Bug fix
- [X] Feature
- [ ] Refactor
 
Describe the feature - fix - refactor:

- Add testing for the NeedMilk.js, MilkRequestForm.js and the ConfirmationMessage.js components.
- Create fixture for testing
- Add classNames to the input fields for MilkRequestForm.js

How was this tested?
- [ ] open index.html
- [x] localhost
- [ ] terminal

Where should the reviewer start - test:

- npm run cypress
- click on 'need_milk_spec.js'

Additional comments:

-  I can split them up but they are one user flow. Don't have sad path yet and wanted to if we are gonna just do that on separate error components etc.

### CODE REVIEW

**Comment by janitastic** [comment]
Hold on merging this one. @cagallo is still making some updates.

File: cypress/integration/need_milk_spec.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+describe('need milk donor results page user flow', () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'https://secret-forest-87730.herokuapp.com/graphql', (req) => {
```
Comment (devonwigle) [inline cypress/integration/need_milk_spec.js:3]: Do we know if BE going to update their url name?

File: cypress/integration/need_milk_spec.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        statusCode: 200,
+        fixture: 'donorMoms.json'
+      })
+    })
+
+    cy.visit('http://localhost:3000/need-milk')
+  })
+
+  it('should have a donor results container that displays three donors', () => {
+    cy.get('.main-container').should('exist')
```
Comment (devonwigle) [inline cypress/integration/need_milk_spec.js:14]: we should be consistent with use of test ids or not

**Review by devonwigle — CHANGES_REQUESTED** [review]

File: cypress/integration/need_milk_spec.js (hunk 1 above)
Comment (cagallo) [reply cypress/integration/need_milk_spec.js:3]: they definitely will, we will need to update tests when they do.

File: cypress/integration/need_milk_spec.js (hunk 2 above)
Comment (cagallo) [reply cypress/integration/need_milk_spec.js:14]: Yeah, I'm gonna add them in, just have never used them so figured I'd add them in after I finished the test!

**Review by lschnell8 — APPROVED** [review]
Thanks Chez! Looks great.

**Pull request merged**